docs/specs/tripgo.swagger.yaml

```diff
@@ -2015,6 +2000,63 @@ definitions:
       - alt
       - localIcon
 
+  ModeIdentifier:
+    description: This is a map/dictionary with multiple items of this type. Each key is a mode identifier. See https://developer.tripgo.com/faq/#mode-identifiers.
+    properties:
+      title:
+        type: string
+      URL:
+        type: string
+      color:
+        $ref: '#/definitions/Color'
+      icon:
+        type: string
+        description: File name part for downloading an image for this mode. [See separate documentation](TODO) for how to construct a URL from this.
+      implies:
+        type: string
+        description: Another mode identifier that is required by this one, e.g., you can't request only school buses without also requesting regular public transport.
+    required:
+      - title
+
+  ModeDetails:
```

Comment:
Can this extend ModeIdentifier?

Reply:
`ModeIndentifier` has some fields we don't have here, like `icon` (we have localIcon) and `implies`

Reply:
I see. Maybe let's review this when we bump the version number to unify the two.